SourceGeneratorTests/Tests.cs

```diff
@@ -28,6 +36,8 @@ void M()
 
             Assert.NotNull(output);
 
+            _output.WriteLine(output);
```

Comment:
Nice!

I'd move this to `GetGeneratedOutput`

Reply:
Moved to `GetGeneratedOutput` and updated method to not be static.